Libraries/dotNetRDF/Query/Builder/TriplePatternObjectPart.cs

```diff
@@ -129,5 +129,15 @@ public ITriplePatternBuilder Object(PatternItem objectPattern)
             _triplePatternBuilder.AddPattern(new TriplePattern(_subjectPatternItem, _predicatePatternItem, objectPattern));
             return _triplePatternBuilder;
         }
+
+        /// <summary>
+        /// Depending on the generic parameter type, sets a literal, a QName or a blank node as <see cref="IMatchTriplePattern.Object"/>
+        /// </summary>
+        /// <param name="objectUri">Either a variable name, a literal, a QName or a blank node identifier</param>
+        /// <remarks>A relevant prefix/base URI must be added to <see cref="IQueryBuilder.Prefixes"/> to accept a QName</remarks>
```

Comment:
Please adjust the XML doc. This method only accepts prefixed names. You might also want to rename the parameter so that it's clear that fully qualified URIs will not work.